crates/bws/src/main.rs

```diff
@@ -89,6 +105,41 @@ enum GetCommand {
     Secret { secret_id: String },
 }
 
+#[derive(Subcommand, Debug)]
+enum CreateCommand {
+    Secret {
+        key: String,
+        value: String,
+
+        /// An optional note to add to the secret
```

Comment:
Should we move this to be a description of the arg instead of a comment?

Reply:
Done